clarifai/client/input.py

```diff
@@ -19,6 +19,7 @@
 from clarifai.client.base import BaseClient
 from clarifai.client.lister import Lister
 from clarifai.constants.dataset import MAX_RETRIES
+from clarifai.constants.rag import MAX_UPLOAD_BATCH_SIZE
```

Comment:
we can add this constant in dataset itself to avoid confusion..
`from clarifai.constants.dataset import MAX_RETRIES, MAX_UPLOAD_BATCH_SIZE `

Reply:
> we can add this constant in dataset itself to avoid confusion.. `from clarifai.constants.dataset import MAX_RETRIES, MAX_UPLOAD_BATCH_SIZE `

Even I thought of same, should we do that or we can add new constant file for input.py ?? 
WDYT

Reply:
Yeah constants for input.py is more relevant 👍